.circleci/config.yml

```diff
@@ -687,6 +697,11 @@ jobs:
             - influxdb-gomod-sum-{{ checksum "go.sum" }}
       - install_core_deps
       - upgrade_go
+      - run:
+          # Goreleaser's blob stanza requires relative paths
```

Comment:
Is this comment referring to the need to copy the artifacts to `./changelog_artifacts`?

Reply:
Yes